cni/network/network.go

```diff
@@ -740,6 +764,8 @@ func (plugin *NetPlugin) createEndpointInternal(opt *createEndpointInternalOpt)
 	}
 
 	if opt.resultV6 != nil {
+		// inject routes to linux pod
+		epInfo.IPV6Mode = network.DualStackOverlay
```

Comment:
why ipv6Mode is concluded as DualstackOverlay and how do we handle if we support podsubnet(vnet) for ipv6 tomorrow?

Reply:
also where we are using this field?

Reply:
we are using this field in network_windows.go:
// only add net rules if it's dualStackOverlay mode and hnsNetwork is created at first time
			if util.DualStackOverlay == DualStackOverlay {
				if err := nm.addNewNetRules(nwInfo); err != nil {
					return nil, err
				}
			}